### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by meistudioli**
typo

Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

- 
- 
-

### CODE REVIEW

File: src/site/content/en/blog/min-max-clamp/index.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ conjunction with `calc()` (i.e. `font-size: max(calc(0.5vw - 1em), 2rem)`),  or
 composed (i.e. `font-size: max(min(0.5vw, 1em), 2rem)`). 
 
 {% Aside %} When using a calculation inside of a `min()`, `max()`, or `clamp()`
-function, you can nix the `calc()`. For example, writing `font-size:
+function, you can mix the `calc()`. For example, writing `font-size:
```
Comment (jpmedley) [inline src/site/content/en/blog/min-max-clamp/index.md:100]: Actually, I think 'nix' is the right word here. For this to make sense, the formulas in the following line need to be reversed.

@una, please weigh in.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/min-max-clamp/index.md (hunk 1 above)
Comment (una) [reply src/site/content/en/blog/min-max-clamp/index.md:100]: nix is correct

Comment (una) [reply src/site/content/en/blog/min-max-clamp/index.md:100]: You can remove the calc() might be more clear?

Comment (kaycebasques) [reply src/site/content/en/blog/min-max-clamp/index.md:100]: "Nix" is technically correct but this PR is evidence that using a less common word causes confusion. So, yes, let's replace.

Comment (kaycebasques) [reply src/site/content/en/blog/min-max-clamp/index.md:100]:
```suggestion
function, you can remove the call to `calc()`. For example, writing `font-size:
```

**Review by kaycebasques — APPROVED** [review]

**Pull request merged**